airflow/models/dag.py

```diff
@@ -227,7 +230,8 @@ def __init__(
         params: Optional[Dict] = None,
         access_control: Optional[Dict] = None,
         is_paused_upon_creation: Optional[bool] = None,
-        jinja_environment_kwargs: Optional[Dict] = None
+        jinja_environment_kwargs: Optional[Dict] = None,
+        tags: Optional[List[str]] = []
```

Comment:
If we use `Optional` should we use `None` as the default value?

Reply:
My bad, fixing.